src/backend/src/services/change-requests.services.ts

```diff
@@ -614,87 +613,93 @@ export default class ChangeRequestsService {
       }
     });
 
-    if (wbsProposedChanges) {
-      if (projectProposedChanges && workPackageProposedChanges) {
-        throw new HttpException(400, "Change Request can't be on both a project and a work package");
-      } else if (!projectProposedChanges && !workPackageProposedChanges) {
-        throw new HttpException(
-          400,
-          'Change Request with proposed changes must have either project or work package proposed changes'
-        );
-      } else {
-        const { name, status, projectLeadId, projectManagerId, links } = wbsProposedChanges;
+    let name: string = '';
+    let status: WbsElementStatus = WbsElementStatus.Active;
+    let projectLeadId: number = 0;
+    let projectManagerId: number = 0;
+    let links: { url: string; linkTypeName: string }[] = [];
 
-        if (projectLeadId) {
-          const projectLead = await prisma.user.findUnique({ where: { userId: projectLeadId } });
-          if (!projectLead) throw new NotFoundException('User', projectLeadId);
-        }
+    if (projectProposedChanges && workPackageProposedChanges) {
+      throw new HttpException(400, "Change Request can't be on both a project and a work package");
+    } else if (!projectProposedChanges && !workPackageProposedChanges) {
+      throw new HttpException(
+        400,
+        'Change Request with proposed changes must have either project or work package proposed changes'
+      );
+    } else if (projectProposedChanges) {
```

Comment:
you're overcomplicating this, from this point, have one else if for projectProposedChanges and do everything you need for that inside of there (validating any inputs that correspond to models in the database, and then creating the wbsProposedChanges and projectProposedChanges in the database, I think you should be able to do it in one prisma call) then have another else if that does the same for workPackageProposedChanges